packages/jupyter-ai/jupyter_ai/personas/persona_manager.py

```diff
@@ -260,3 +268,116 @@ def get_chat_dir(self) -> str:
         """
         abspath = self.get_chat_path(absolute=True)
         return os.path.dirname(abspath)
+
+
+class LocalPersonaLoader(LoggingConfigurable):
+    """
+    Load _persona class declarations_ from the local filesystem.
+
+    Those class declarations are then used to instantiate personas by the `PersonaManager`.
+
+    TODO: wire to local .jupyter directory system, for now just use the current
+    directory.
+    """
+
+    def __init__(
```

Comment:
This will all go away, but we usually avoid using an __init__ method with LoggingConfigurable. If we need to keep the class we can share examples, but my sense is to switch to a function.